services/simcore/docker-compose.yml

```diff
@@ -532,6 +532,34 @@ services:
           memory: 128M
           cpus: '0.1'
 
+  efs-guardian:
+    networks:
+      - monitored
+      - public
+    hostname: "{{.Service.Name}}"
+    deploy:
+      replicas: ${SIMCORE_EFS_GUARDIAN_REPLICAS}
+      update_config:
+        parallelism: 2
+        order: start-first
+        failure_action: continue
+        delay: 10s
+      restart_policy:
+        condition: any
+        delay: 5s
+        max_attempts: 3
+        window: 120s
+      placement:
+        constraints:
+          - node.labels.simcore==true
+      resources:
+        limits:
+          memory: 256M
+          cpus: '0.5'
```

Comment:
just fyi: this might lead to your service being throttled. There is a prometheus metric to detect throttling, let me know if you care